[feature][blog] Add 292 release blog (#15164)
apache/pulsar: site2/website/blog/2022-04-08-Apache-Pulsar-2-9-2.md

# Notable bug fixes and enhancements

### Readers continue to read data from the compacted ledgers. [PR-13629](https://github.com/apache/pulsar/pull/13629)

#### Issue

Previously, when topics were unloaded, some data was lost to be read by readers if they have consumed some messages from some compacted ledgers.

#### Resolution

Rewound the reader cursor to the next message of the mark delete position if `readCompacted = true`.

### Brokers decrease the number of unacked messages. [PR-13383](https://github.com/apache/pulsar/pull/13383)

#### Issue

Previously, brokers did not decrease the number of unacked messages if batch ack was enabled. Consequently, consumers were blocked if they reached `maxUnackedMessagesPerConsumer` limit.

#### Resolution

Decreased the number of unacked messages when `individualAckNormal` was called.

### Chunked messages can be queried through Pulsar SQL. [PR-12720](https://github.com/apache/pulsar/pull/12720)

#### Issue

Previously, chunked messages could not be queried through Pulsar SQL.

#### Resolution

> Added a chunked map to support this enhancement.

Reviewer: ```suggestion
Add a chunked message map in PulsarRecordCursor to maintain incomplete chunked messages, if one chunked message was received completely, it will be offered in the message queue to wait for deserialization. 
```

### Support enable or disable schema upload at the broker level. [PR-12786](https://github.com/apache/pulsar/pull/12786)

#### Issue

> Previously, schemas could be automatically uploaded only by changing namespace policies.

Reviewer: ```suggestion
Previously, Pulsar didn't support enabling or disabling schema upload at the broker level.
```

#### Issue

Previously, readers were not able to read the latest messages in compacted topics if readers enabled `readCompacted` and all the data of topics has been compacted to compacted legers.

#### Resolution

Added the `forceReset` configuration for the managed cursor, so that the cursor could be reset to a given position and readers can read data from compacted ledgers. 

### Transaction sequenceId can be recovered correctly. [PR-13209](https://github.com/apache/pulsar/pull/13209)

#### Issue

Previously, the wrong transaction sequenceId was recovered due to incorrect `managedLedger` properties.

#### Resolution

Used `ManagedLedgerInterceptor ` to update current sequenceId to `managedLedger` properties and more.

### Transactions performance test tool is available. [PR-11933](https://github.com/apache/pulsar/pull/11933)

#### Issue

Previously, it was hard to test transaction performance (such as the delay and rate of sending and consuming messages) when opening a transaction.

#### Resolution

Added `PerformanceTransaction` class to support this enhancement.

> ### No race condition exists in Pulsar proxy. [PR-14078](https://github.com/apache/pulsar/pull/14078)

Reviewer: ```suggestion
### Port exhaustion and connection issues in Pulsar Proxy. [PR-14078](https://github.com/apache/pulsar/pull/14078)
```

#### Issue

> Previously, proxy stopped working for proxying broker connections while Admin API proxying kept working.

Reviewer: ```suggestion
Previously, the Pulsar proxy can get into a state where it stops proxying Broker connections while Admin API proxying keeps working.
```
